example/discord.js

```diff
@@ -29,3 +28,6 @@ client.on('message', message => {
       });
   }
 });
+
+
+client.login(clientToken);
```

Comment:
Why move `.login()` to end when it's asynchronous?

Reply:
its more human understandble